task10.5.1.js

```diff
@@ -0,0 +1,20 @@
+// Есть массив [ ‘Tony’, ‘Stark’,  1 , 45, 2, 5, 34, 9, 11]
+// Присвоить первое и второе значения массива к соответствующим переменным, а остальные значения сложить 
+
+let arr = ['Tony', 'Stark', 1, 45, 2, 5, 34, 9, 11];
+let obj = {};
+let func = (x) => {
+    let sum = 0;
```

Comment:
It's better to use rest operator for numbers and set first and second item of the array as variables with help destructurization